src/OpenTelemetry.Instrumentation.Runtime/RuntimeMetrics.cs

```diff
@@ -99,6 +100,12 @@ public RuntimeMetrics(RuntimeMetricsOptions options)
             {
                 this.meter.CreateObservableCounter($"{metricPrefix}assembly.count", () => (long)AppDomain.CurrentDomain.GetAssemblies().Length, description: "Number of Assemblies Loaded.");
             }
+
+            if (options.IsExceptionsEnabled)
+            {
+                AppDomain.CurrentDomain.FirstChanceException += (source, e) => ++this.exceptionCount;
```

Comment:
need threadsafe addition for exceptionCount

Reply:
Changed to Interlocked.Increment method.